starwars_api/models.py

```diff
@@ -11,15 +11,21 @@ def __init__(self, json_data):
         Dynamically assign all attributes in `json_data` as instance
         attributes of the Model.
         """
-        pass
+        for key, value in json_data.items():
+            setattr(self, key, value)
+            # setattr(x, 'foobar', 123) is equivalent to x.foobar = 123
```

Comment:
@StefGou glad to help! I included the comment cos I didn't know `setattr(self, key, value)` was a thing and wondered _how_ it made sense. i want to revisit my class work after the course so I didn't want to re-do all my googling XD